### ISSUE TITLE
Feedback privacy check

### ISSUE THREAD

**Issue by sheepyhollow** [zentrumnawi/solid-frontend#1559]

**Comment by sheepyhollow** [comment]
Können wir die (app-spezifische) `privacy.component.html` auch in einem Popup öffnen?

**Comment by sheepyhollow** [comment]
Ich habe probiert eine `mat-checkbox` in das Feedback-Form einzufügen und es funktioniert einfach nicht 🤷
Die Checkbox wird nicht angezeigt, auch, wenn ich sie außerhalb von <form> hinzufüge...

**Comment by sheepyhollow** [comment]
> Ich habe probiert eine `mat-checkbox` in das Feedback-Form einzufügen und es funktioniert einfach nicht

duh... Man muss erst das `MatCheckboxModule` in `solid-keleton.module.ts` importieren

**Comment by lthuong** [comment]
> Können wir die (app-spezifische) `privacy.component.html` auch in einem Popup öffnen?

Wollen wir das auch beim Feedback hinzufügen ? Ich muss es kurz anschauen ob das machbar ist 🤔

**Comment by sheepyhollow** [comment]
Ja, genau. Es geht darum, dass ich die Datenschutzerklärung lesen will, wenn ich gerade das Feedback offen habe.

Da wäre ein Dialog/Popup gut. Sonst muss ich ja das Feedback schließen.

(Wäre alles nicht so schlimm, weil das Feedback zwischengespeichert wird und die meisten das privacy-Dings sowieso nicht lesen, ABER... you know...)

**Comment by sheepyhollow** [comment]
> Ich muss kurz überlegen wie ich das implementieren soll

Über den Feedback-Dialog öffnet sich noch ein Dialog mit dem Content von privacy (so ähnlich wie in diesem [Beispiel](https://material.angular.io/components/dialog/examples#dialog-content))

**Comment by sheepyhollow** [comment]
Ist in 77766e959e30b45dbe4155ae4a62d5d3286290ae eingebaut, aber ich habe noch ein styling/overflow-Problem.

> Können wir die (app-spezifische) privacy.component.html auch in einem Popup öffnen?

hier wäre die Idee, das Wort `Datenschutzerklärung` als Link zu haben und damit das Popup zu öffnen.
